.github/workflows/ubuntu_clean_meson_build.yml

```diff
@@ -28,16 +28,75 @@ jobs:
       if: env.rebuild == '1'
       run: |
         sudo apt-get update && sudo apt-get install -y libglib2.0-dev libjson-glib-dev libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev libunwind-dev googletest gdb
-        sudo apt-get install -y liborc-0.4-dev flex bison libopencv-dev pkg-config python3-dev python3-numpy python3
+        sudo apt-get install -y liborc-0.4-dev flex bison libopencv-dev pkg-config python3-dev python3-numpy python3 libcurl4-openssl-dev
         sudo add-apt-repository -y ppa:nnstreamer/ppa && sudo apt-get update && sudo apt-get install -y ssat libpaho-mqtt-dev
         sudo apt-get install -y valgrind gstreamer1.0-tools gstreamer1.0-plugins-good gstreamer1.0-plugins-base libgtest-dev libpng-dev libc6-dbg binutils-x86-64-linux-gnu-dbg valgrind-dbg
         pip install meson ninja
+    - name: install llama.cpp pre-built binaries
+      if: env.rebuild == '1' && matrix.os == 'ubuntu-22.04'
+      run: |
+        #Way to use pre-built
+        # We need to copy the ggml backends to the location of the llama.cpp TC executable
+        # In the ggml-org/ggml#1120 issue, the following issues were identified:
+        # 1. The requirement to extend GGML's dynamic backend search logic to comply with the Linux Filesystem Hierarchy Standard (FHS).
+        # 2. The requirement to have a separate developer package (dev package) containing header files, pkg-config, and CMake configuration files.
+        # 3. The requirement to enable backend search in the system standard library paths (/usr/lib, /lib) in addition to the executable path.
+
+        export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+        mkdir -p ./tests/test_models/models/
+        wget https://huggingface.co/TheBloke/TinyLlama-1.1B-Chat-v1.0-GGUF/resolve/main/tinyllama-1.1b-chat-v1.0.Q2_K.gguf -O ./tests/test_models/models/tinyllama-1.1b-chat-v1.0.Q2_K.gguf
```

Comment:
1.1B is still too big for unit testing. There are under 1MB llama models. Unit test time is already too long.

Refer: https://huggingface.co/delphi-suite/stories-llama2-50k